### PR TITLE
gradual improvement

### PR DESCRIPTION
**Pull request by KedoKudo**

### CODE REVIEW

File: ipython_profiles/profile_s6bm/startup/03-plans.py [hunk 1]
```diff
@@ -140,6 +141,7 @@ def scan_closure():
                 yield from bps.mv(preci, ang)
                 yield from bps.trigger_and_read([det])
         elif config['tomo']['type'].lower() == 'fly':
+            yield from bps.mv(det.proc1.num_filter, 1)
```
Comment (prjemian) [inline ipython_profiles/profile_s6bm/startup/03-plans.py:144]: Would any sequencing be lost if these three lines were combined into a single `yield from bps.mv(...)`?

File: ipython_profiles/profile_s6bm/startup/03-plans.py [hunk 2]
```diff
@@ -183,6 +185,7 @@ def scan_closure():
         yield from bps.mv(det.cam.frame_type, 2)
 
         # 1-8 take the back white
+        yield from bps.mv(det.proc1.num_filter, n_frames)
```
Comment (prjemian) [inline ipython_profiles/profile_s6bm/startup/03-plans.py:188]: Similar, can these two lines be combined?

File: ipython_profiles/profile_s6bm/startup/03-plans.py [hunk 3]
```diff
@@ -218,3 +221,13 @@ def repeat_exp(plan_func, n=1):
 
 
 print(f'leaving {__file__}...\n')
+
+
+def repeat(n):
```
Comment (prjemian) [inline ipython_profiles/profile_s6bm/startup/03-plans.py:226]: Suggest replacing this code with [`bluesky.plan_stubs.repeat()`](https://github.com/bluesky/bluesky/blob/59773b474fa7a5688bea3582313dfe2946ee3a49/bluesky/plan_stubs.py#L955)

**Review by prjemian — COMMENTED** [review]
Left suggestions for minor changes in code, not algorithm

File: ipython_profiles/profile_s6bm/startup/03-plans.py (hunk 1 above)
Comment (KedoKudo) [reply ipython_profiles/profile_s6bm/startup/03-plans.py:144]: Based on previous testings, the sequence of setting each parameters matters and we are in the process of catching all the hidden _tribal knowledge_ through trial and error.  Hence the verbose syntax used in the current version.

File: ipython_profiles/profile_s6bm/startup/03-plans.py (hunk 2 above)
Comment (KedoKudo) [reply ipython_profiles/profile_s6bm/startup/03-plans.py:188]: A more compact/concise syntax will be used once we are confident about the required sequence for setting different parameters.

File: ipython_profiles/profile_s6bm/startup/03-plans.py (hunk 3 above)
Comment (KedoKudo) [reply ipython_profiles/profile_s6bm/startup/03-plans.py:226]: This `repeat()` function is really neat for repeating single scan.  However, is it possible to extend its capability by adding a `timeout` option?

**Pull request merged**